sormas-backend/src/main/java/de/symeda/sormas/backend/common/CoreAdo.java

```diff
@@ -30,15 +34,37 @@
 public class CoreAdo extends DeletableAdo {
 
     public static final String ARCHIVED = "archived";
+	public static final String END_OF_PROCESSING_DATE = "endOfProcessingDate";
+	public static final String ARCHIVE_UNDONE_REASON = "archiveUndoneReason";
 
-    private boolean archived;
+	private boolean archived;
+	private Timestamp endOfProcessingDate;
+	private String archiveUndoneReason;
 
-    @Column(nullable = false)
-    public boolean isArchived() {
-        return archived;
-    }
+	@Column(nullable = false)
+	public boolean isArchived() {
+		return archived;
+	}
 
     public void setArchived(boolean archived) {
         this.archived = archived;
     }
+
+	@Column
+	public Timestamp getEndOfProcessingDate() {
+		return endOfProcessingDate;
+	}
+
+	public void setEndOfProcessingDate(Timestamp endOfProcessingDate) {
+		this.endOfProcessingDate = endOfProcessingDate;
+	}
+
+	@Column(length = CHARACTER_LIMIT_DEFAULT)
```

Comment:
Not sure whether default is sufficient here or might be too short; @MartinWahnschaffe What do you think?